qemu/tests/watchdog.py

```diff
@@ -283,8 +284,14 @@ def hotplug_unplug_watchdog_device():
                 test.fail("Can find watchdog pci")
 
         plug_watchdog_device = params.get("plug_watchdog_device", "i6300esb")
+        machine_type = params.get("machine_type")
         watchdog_device_add = ("device_add driver=%s, id=%s"
                                % (plug_watchdog_device, "watchdog"))
+        if machine_type == "q35":
+            bus = "pcie-pci-bridge-0"
+            if not vm.devices.has_device('pcie-pci-bridge'):
+                bus = "pci-bridge-0"
+            watchdog_device_add += ",bus=%s,addr=0x1f" % bus
```

Comment:
Only one watchdog hotplug case, no need to add common function in framework. And `addr=0x1f` is safe.

Acked-by: Yanan Fu <yfu@redhat.com>